packages/react-components/react-rating-preview/src/components/Rating/renderRating.tsx

```diff
@@ -0,0 +1,16 @@
+/** @jsxRuntime classic */
+/** @jsx createElement */
+
+import { createElement } from '@fluentui/react-jsx-runtime';
```

Comment:
after this lands https://github.com/microsoft/fluentui/pull/29214/files it might be good idea to re-generate the package or manually update render files